### DISCUSSION TITLE
Sharing my RobocodeTankRoyale-WEB-GUI

### DISCUSSION THREAD

**Discussion by SirStone** [robocode-dev/tank-royale#49] [General]
Hi
I'm happy(scared) to say that I've done some work on my alternative UI project in order to make it usable by others, created a long and boring README file and put all on my github page.

Here a comfortable link: [RobocodeTankRoyale-alternative-Web-GUI](https://github.com/SirStone/RobocodeTankRoyale-alternative-Web-GUI)

@Steven Tobias aka stobias123 thanks for the request you've done, nothing pushed me more than that in putting the repo available to public.

**Comment by flemming-n-larsen** [comment]
That is really great news! And congratulations with this achievement. 😀
I hope people will find it useful.

I will try this out very soon!

**Reply by flemming-n-larsen** [reply]
I guess I should provide the server and booter jar as independent files as well, e.g. on a public Maven repository or with the release.

**Reply by SirStone** [reply]
I think you are guessing right, if on of your targets is provide maximum modularity you should, I've always thought that was weird that those packages were not available right away.

**Comment by Arne-B** [comment]
Nice, i only found this now... need to check it out! I recently started a browser based observer client... will publish the current state somewhere soon.
